dotcom-rendering/src/web/components/LiveBlock.tsx

```diff
@@ -111,7 +111,7 @@ export const LiveBlock = ({
 					block.blockLastUpdated &&
 					block.blockLastUpdatedDisplay && (
 						<LastUpdated
-							lastUpdated={block.blockLastUpdated}
+							lastUpdated={new Date(block.blockLastUpdated)}
 							lastUpdatedDisplay={block.blockLastUpdatedDisplay}
```

Comment:
Are we saying that we will always replace this string with a new 'local' value on the client? I wonder if it's better to just not render anything on the server at all. That way we avoid the flash of incorrect content which could be very distracting when trying to read a post (it could well look as though the post has _just_ updated when in fact we just changed the time format)

Reply:
The change to this component is only for the prop type because we thought this could simplify things. Inside LastUpdated we convert the `number` to a `Date`. In apps-rendering the block lastUpdated property is a `Date`, so we had to convert to a `number` only for it to be converted back to a `Date` again.

Reply:
I was more thinking about the fact that elsewhere in this PR we are adding a client side action to mutate the value of this dom entry. This means the html that is created on the server and sent to the client will change. That is, it will get rendered on the page as one thing, say,'15th Dec 10:00 GMT', and then, after a short delay, will suddenly show as '15th Dec 05:00 EST'.

To avoid that flash of incorrect content, it might be better to not render the value on the server?

Reply:
We discussed this with Jamie - we expect the overall date time string to vary by max 1 character (timezones can be either 3 or 4 chars in length). As such, the possible CLS resulting from this might be quite small. Also the client side bundle has been known to break and, given the possible legal requirements to show last updated time, it was decided to render at least the server side time.